File: app/src/main/java/com/oddlyspaced/calci/mathtype/enums/MTOperandSide.java
package com.oddlyspaced.calci.mathtype.enums;
/* loaded from: classes.dex */
public enum MTOperandSide {
    Left,
    Right;

    public static MTOperandSide opposite(MTOperandSide operandSide) {
        return operandSide == Left ? Right : Left;
    }
}

File: app/src/main/java/com/oddlyspaced/calci/mathtype/enums/MTOperandSide.kt
package com.oddlyspaced.calci.mathtype.enums

enum class MTOperandSide {
    Left,
    Right;

    companion object {
        fun opposite(operandSide: MTOperandSide): MTOperandSide {
            return if (operandSide == Left) Right else Left
        }
    }
}